src/Arcus.EventGrid.Tests.Unit/Parsing/AzureIoTHubParsingTests.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Linq;
+using Arcus.EventGrid.IoTHub.Contracts.Events.v1;
+using Arcus.EventGrid.Parsers;
+using Arcus.EventGrid.Tests.Unit.Artifacts;
+using Xunit;
+
+namespace Arcus.EventGrid.Tests.Unit.Parsing
+{
+    public class AzureIoTHubParsingTests
+    {
+        [Fact]
+        public void Parse_ValidDeviceCreatedEvent_ShouldSucceed2()
```

Comment:
Maybe for maintainability reasons, you can assert on the entire event by (for example) both serializing the expected and actual event.

Reply:
We already assert on the actual event which is the goal of this test, we can add another test which creates an instance of this event and assert it as well, but would create an issue for that